vendor/github.com/openshift/library-go/pkg/manifestclient/list.go

```diff
@@ -241,7 +241,7 @@ func allPossibleListFileLocations(sourceFS fs.FS, requestInfo *apirequest.Reques
 
 		namespaces, err := allNamespacesWithData(sourceFS)
 		if err != nil {
-			return nil, fmt.Errorf("unable to read namespaces: %w", err)
+			return allPossibleListFileLocations, fmt.Errorf("unable to read namespaces: %w", err)
```

Comment:
it seems that requiring  `namespaces` is not necessary when the input contains only `cluster-scoped-resources` (my test), wdyt?

Reply:
xref: https://github.com/openshift/library-go/pull/1888